tests/unit/test_proxy_minion.py

```diff
@@ -27,56 +29,88 @@ def test_post_master_init_metaproxy_called(self):
+        mock_opts.update(salt.config.DEFAULT_PROXY_MINION_OPTS)
+
         mock_data = {"fun": "foo.bar", "jid": 123}
         mock_jid_queue = [123]
         proxy_minion = salt.minion.ProxyMinion(
             mock_opts,
             jid_queue=copy.copy(mock_jid_queue),
             io_loop=salt.ext.tornado.ioloop.IOLoop(),
         )
-        try:
-            ret = proxy_minion._handle_decoded_payload(mock_data).result()
-            self.assertEqual(proxy_minion.jid_queue, mock_jid_queue)
-            self.assertIsNone(ret)
-            self.assert_called_once(salt.minion._metaproxy_call)
-        finally:
-            proxy_minion.destroy()
+        mock_metaproxy_call = MagicMock()
+        with patch(
+            "salt.minion._metaproxy_call",
+            return_value=mock_metaproxy_call,
+            autospec=True,
+        ):
+            try:
+                ret = proxy_minion._handle_decoded_payload(mock_data).result()
+                self.assertEqual(proxy_minion.jid_queue, mock_jid_queue)
+                self.assert_called_once(salt.minion._metaproxy_call)
+            finally:
+                proxy_minion.destroy()
 
     @slowTest
     def test_handle_payload_metaproxy_called(self):
         """
         Tests that when the _handle_payload function is called, _metaproxy_call is also called.
         """
         mock_opts = salt.config.DEFAULT_MINION_OPTS.copy()
+        mock_opts.update(salt.config.DEFAULT_PROXY_MINION_OPTS)
+
         mock_data = {"fun": "foo.bar", "jid": 123}
         mock_jid_queue = [123]
         proxy_minion = salt.minion.ProxyMinion(
             mock_opts,
             jid_queue=copy.copy(mock_jid_queue),
             io_loop=salt.ext.tornado.ioloop.IOLoop(),
         )
-        try:
-            ret = proxy_minion._handle_decoded_payload(mock_data).result()
-            self.assertEqual(proxy_minion.jid_queue, mock_jid_queue)
-            self.assertIsNone(ret)
-            self.assert_called_once(salt.minion._metaproxy_call)
-        finally:
-            proxy_minion.destroy()
+        mock_metaproxy_call = MagicMock()
+        with patch(
+            "salt.minion._metaproxy_call",
+            return_value=mock_metaproxy_call,
+            autospec=True,
+        ):
+            try:
+                ret = proxy_minion._handle_decoded_payload(mock_data).result()
+                self.assertEqual(proxy_minion.jid_queue, mock_jid_queue)
+                self.assert_called_once(mock_metaproxy_call)
+            finally:
+                proxy_minion.destroy()
+
+    def test_proxy_config_default_include(self):
+        """
+        Tests that when the proxy_config function is called,
+        for the proxy minion, eg. /etc/salt/proxy.d/dummy/*.conf
+        """
+        opts = salt.config.proxy_config(
+            RUNTIME_VARS.TMP_PROXY_CONF_DIR + "/proxy", minion_id="proxytest"
+        )
+        self.assertIn("schedule", opts)
+        self.assertIn("test_job", opts["schedule"])
```

Comment:
Didn't I commit something to this PR wich addressed this test?
It's broken now.

Reply:
I did and it got merged. Nevermind.